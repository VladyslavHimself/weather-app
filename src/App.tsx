const App = () => {
    return <div className="App" />;
};

export default App;
