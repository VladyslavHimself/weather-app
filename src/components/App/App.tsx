import logoSvg from '../../assets/logotype.svg';

function App() {
    return (
        <div className="App">
            <div className="navbar">
                <div className="logotype">
                    <img src={logoSvg} alt="logotype" />
                </div>
                <div className="search-field">s</div>
                <div className="metrics">s</div>
            </div>
        </div>
    );
}

export default App;
