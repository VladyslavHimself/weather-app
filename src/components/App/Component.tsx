import 'normalize.css';
import { useState } from 'react';
import logoSvg from '../../assets/logotype.svg';
import ChooseMetricButton from '../ChooseMetricButton/Component';
import SearchField from '../SearchField/Component';

import classes from './styles.module.scss';

function App() {
    const [first, setfirst] = useState<string>('');

    return (
        <div className="App">
            <div className={classes.navbar}>
                <div className={classes.container}>
                    <div className={classes.logotype}>
                        <img src={logoSvg} alt="logotype" />
                    </div>
                    <div className={classes.search}>
                        <SearchField
                            stateValue={first}
                            setStateValue={setfirst}
                        />
                    </div>
                    <ChooseMetricButton />
                </div>
            </div>
        </div>
    );
}

export default App;
