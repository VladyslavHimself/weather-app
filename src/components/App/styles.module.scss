@import '../../styles/variables';
@import '../../styles/colors';

.today-weather {
    width: 100%;
    min-height: 405px;

    display: flex;
    align-items: center;
    justify-content: center;

    .container {
        width: $container-max-width;
        display: flex;
        justify-content: space-between;
        flex-wrap: wrap;
    }
}

.average-data {
    width: 256px;

    display: flex;

    font-family: 'Roboto';
    font-style: normal;
    flex-direction: column;

    &__city {
        font-weight: 400;
        font-size: 32px;
        line-height: 38px;
    }

    &__time {
        font-weight: 400;
        font-size: 18px;
        line-height: 21px;
        color: $text-color-secondary;
    }

    &__temperature {
        font-style: normal;
        font-weight: 500;
        font-size: 96px;
        line-height: 112px;
        color: $text-color;
        margin-top: 26px;
    }
}

.general-data {
    width: 837px;
    display: flex;
    justify-content: space-around;
    flex-wrap: wrap;
}

.data-item {
    width: 87px;
    display: flex;
    justify-content: center;
    align-items: center;
    flex-direction: column;

    font-family: 'Roboto';
    font-style: normal;

    &__time {
        font-weight: 400;
        font-size: 32px;
        line-height: 38px;
        color: $text-color-secondary;
    }

    &__weather-type {
        margin: 20px 0px 25px 0px;
    }

    &__temperature {
        font-weight: 400;
        font-size: 32px;
        line-height: 38px;
    }
}

.detailed-now-weather {
    position: relative;
    width: 100%;

    display: flex;
    justify-content: space-around;
    flex-wrap: wrap;

    background-color: $background-inverse;

    font-family: 'Roboto', sans-serif;

    &__hint {
        position: absolute;
        top: 10%;
        left: 1%;
        transform: translate(-1%, -10%);

        color: $text-color-inverse;

        font-weight: 700;
        font-size: 18px;
        line-height: 21px;
    }

    .container {
        width: $container-max-width;
        min-height: 178px;
        display: flex;
        justify-content: space-between;
        align-items: center;
    }

    .detailed-information-tab {
        width: 121px;
        height: 64px;

        display: flex;
        justify-content: center;
        align-items: center;
        flex-direction: column;

        color: $text-color-inverse;

        // border: 1px solid red;
        &__title {
            font-style: normal;
            font-weight: 400;
            font-size: 18px;
            line-height: 21px;
        }

        &__data {
            margin-top: 15px;

            font-weight: 700;
            font-size: 24px;
            line-height: 28px;
        }
    }
}

@media (max-width: 1092px) {
    .today-weather {
        .container {
            width: 90%;
            justify-content: center;
            .average-data {
                margin-top: 50px;
            }
            .general-data {
                margin-top: 20px;
            }
        }
    }
}

@media (max-width: 575px) {
    .today-weather {
        .container {
            width: 90%;
            justify-content: center;
            .general-data {
                flex-direction: column;
                align-items: center;

                .data-item {
                    margin: 25px 0;
                }
            }
        }
    }
}

// detailed-now-weather

@media (max-width: 725px) {
    .detailed-now-weather {
        &__hint {
            top: 5%;
            left: 50%;

            transform: translate(-50%, -5%);
        }
        .container {
            flex-direction: column;
            margin: 100px 0;
        }
    }

    .detailed-information-tab {
        margin: 25px 0;
    }
}
