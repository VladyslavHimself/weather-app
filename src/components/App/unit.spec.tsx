import { render } from '@testing-library/react';
import App from './Component';

describe('Render App components', () => {
    const components = {
        navbar: '.navbar',
        dailyWeather: '.daily-weather',
        hourlyWeather: '.hourly-weather',
        detailedDailyWeather: '.detailed-daily-weather',
    };

    it('should render App correctly', () => {
        const { container } = render(<App />);
        expect(container).toBeInTheDocument();
    });

    it('should render the Navbar component', () => {
        const { container } = render(<App />);
        expect(container).toMatchSnapshot();
        expect(container.querySelector(components.navbar)).toBeInTheDocument();
    });

    it('should render the dailyWeather component', () => {
        const { container } = render(<App />);
        expect(container).toMatchSnapshot();
        expect(
            container.querySelector(components.dailyWeather)
        ).toBeInTheDocument();
    });

    it('should render hourlyWeather component', () => {
        const { container } = render(<App />);
        expect(container).toMatchSnapshot();
        expect(
            container.querySelector(components.hourlyWeather)
        ).toBeInTheDocument();
    });

    it('should render DetailedDailyDescription component', () => {
        const { container } = render(<App />);
        expect(container).toMatchSnapshot();
        expect(
            container.querySelector(components.detailedDailyWeather)
        ).toBeInTheDocument();
    });
});
