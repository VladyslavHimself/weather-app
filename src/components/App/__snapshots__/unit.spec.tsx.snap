// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render App components should render DetailedDailyDescription component 1`] = `
<div>
  <div
    class="App"
  >
    <div
      class="navbar"
    >
      <div
        class="container"
      >
        <div
          class="logotype"
        >
          <img
            alt="logotype"
            src="logotype.svg"
          />
        </div>
        <div
          class="search"
        >
          <div
            class="search-field"
          >
            <input
              class="search-bar"
              type="text"
              value=""
            />
            <button
              class="button"
              type="button"
            >
              <img
                alt="loopa"
                src="loopa.svg"
              />
            </button>
          </div>
        </div>
        <button
          class="metrics"
          type="button"
        >
          <span
            class="active"
          >
            °C
          </span>
          <span
            style="margin: 0px 5px;"
          >
            /
          </span>
          <span>
            °F
          </span>
        </button>
      </div>
    </div>
    <div
      class="today-weather"
    >
      <div
        class="container"
      >
        <div
          class="average-data"
        >
          <div
            class="average-data__city"
          />
          <div
            class="average-data__time"
          >
            undefined undefined
          </div>
          <div
            class="average-data__temperature"
          >
            °C
          </div>
        </div>
        <div
          class="general-data"
        >
          <div
            class="data-item"
          >
            <div
              class="data-item__time"
            >
              08:00
            </div>
            <div
              class="data-item__weather-type"
            >
              <img
                alt="cloudy"
                src="cloud.svg"
              />
            </div>
            <div
              class="data-item__temperature"
            >
              13 °C
            </div>
          </div>
          <div
            class="data-item"
          >
            <div
              class="data-item__time"
            >
              09:00
            </div>
            <div
              class="data-item__weather-type"
            >
              <img
                alt="cloudy"
                src="cloud.svg"
              />
            </div>
            <div
              class="data-item__temperature"
            >
              13 °C
            </div>
          </div>
          <div
            class="data-item"
          >
            <div
              class="data-item__time"
            >
              10:00
            </div>
            <div
              class="data-item__weather-type"
            >
              <img
                alt="cloudy"
                src="cloud.svg"
              />
            </div>
            <div
              class="data-item__temperature"
            >
              13 °C
            </div>
          </div>
          <div
            class="data-item"
          >
            <div
              class="data-item__time"
            >
              11:00
            </div>
            <div
              class="data-item__weather-type"
            >
              <img
                alt="cloudy"
                src="cloud.svg"
              />
            </div>
            <div
              class="data-item__temperature"
            >
              13 °C
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="detailed-now-weather"
    >
      <div
        class="detailed-now-weather__hint"
      >
        Now
      </div>
      <div
        class="container"
      >
        <div
          class="detailed-information-tab"
        >
          <div
            class="detailed-information-tab__title"
          >
            Chance of Rain
          </div>
          <div
            class="detailed-information-tab__data"
          >
            12%
          </div>
        </div>
        <div
          class="detailed-information-tab"
        >
          <div
            class="detailed-information-tab__title"
          >
            Humidity
          </div>
          <div
            class="detailed-information-tab__data"
          >
            60%
          </div>
        </div>
        <div
          class="detailed-information-tab"
        >
          <div
            class="detailed-information-tab__title"
          >
            Wind speed
          </div>
          <div
            class="detailed-information-tab__data"
          >
            10 k/h
          </div>
        </div>
        <div
          class="detailed-information-tab"
        >
          <div
            class="detailed-information-tab__title"
          >
            Visibility
          </div>
          <div
            class="detailed-information-tab__data"
          >
            100km
          </div>
        </div>
        <div
          class="detailed-information-tab"
        >
          <div
            class="detailed-information-tab__title"
          >
            Pressure
          </div>
          <div
            class="detailed-information-tab__data"
          >
            1015 hPa
          </div>
        </div>
      </div>
    </div>
    <div
      class="other-days-weather"
    >
      <div
        class="container"
      >
        <div
          class="weather-table"
        >
          <div
            class="weather-table-item--title"
          >
            <div
              class="weather-table-item__container"
            >
              <div
                class="weather-table-item__data-title"
              >
                Day
              </div>
              <div
                class="weather-table-item__data-title"
              >
                Chance of rain
              </div>
              <div
                class="weather-table-item__data-title"
              >
                Humidity
              </div>
              <div
                class="weather-table-item__data-title"
              >
                Wind
              </div>
              <div
                class="weather-table-item__data-title"
              >
                Temperature
              </div>
            </div>
            <hr />
          </div>
          <div
            class="weather-table-item"
          >
            <div
              class="weather-table-item__container"
            >
              <div
                class="weather-table-item__data-title"
              >
                Mon
              </div>
              <div
                class="weather-table-item__data-title"
              >
                9%
              </div>
              <div
                class="weather-table-item__data-title"
              >
                57%
              </div>
              <div
                class="weather-table-item__data-title"
              >
                10 km/h
              </div>
              <div
                class="weather-table-item__data-title"
              >
                19˚
              </div>
            </div>
            <hr />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Render App components should render hourlyWeather component 1`] = `
<div>
  <div
    class="App"
  >
    <div
      class="navbar"
    >
      <div
        class="container"
      >
        <div
          class="logotype"
        >
          <img
            alt="logotype"
            src="logotype.svg"
          />
        </div>
        <div
          class="search"
        >
          <div
            class="search-field"
          >
            <input
              class="search-bar"
              type="text"
              value=""
            />
            <button
              class="button"
              type="button"
            >
              <img
                alt="loopa"
                src="loopa.svg"
              />
            </button>
          </div>
        </div>
        <button
          class="metrics"
          type="button"
        >
          <span
            class="active"
          >
            °C
          </span>
          <span
            style="margin: 0px 5px;"
          >
            /
          </span>
          <span>
            °F
          </span>
        </button>
      </div>
    </div>
    <div
      class="today-weather"
    >
      <div
        class="container"
      >
        <div
          class="average-data"
        >
          <div
            class="average-data__city"
          />
          <div
            class="average-data__time"
          >
            undefined undefined
          </div>
          <div
            class="average-data__temperature"
          >
            °C
          </div>
        </div>
        <div
          class="general-data"
        >
          <div
            class="data-item"
          >
            <div
              class="data-item__time"
            >
              08:00
            </div>
            <div
              class="data-item__weather-type"
            >
              <img
                alt="cloudy"
                src="cloud.svg"
              />
            </div>
            <div
              class="data-item__temperature"
            >
              13 °C
            </div>
          </div>
          <div
            class="data-item"
          >
            <div
              class="data-item__time"
            >
              09:00
            </div>
            <div
              class="data-item__weather-type"
            >
              <img
                alt="cloudy"
                src="cloud.svg"
              />
            </div>
            <div
              class="data-item__temperature"
            >
              13 °C
            </div>
          </div>
          <div
            class="data-item"
          >
            <div
              class="data-item__time"
            >
              10:00
            </div>
            <div
              class="data-item__weather-type"
            >
              <img
                alt="cloudy"
                src="cloud.svg"
              />
            </div>
            <div
              class="data-item__temperature"
            >
              13 °C
            </div>
          </div>
          <div
            class="data-item"
          >
            <div
              class="data-item__time"
            >
              11:00
            </div>
            <div
              class="data-item__weather-type"
            >
              <img
                alt="cloudy"
                src="cloud.svg"
              />
            </div>
            <div
              class="data-item__temperature"
            >
              13 °C
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="detailed-now-weather"
    >
      <div
        class="detailed-now-weather__hint"
      >
        Now
      </div>
      <div
        class="container"
      >
        <div
          class="detailed-information-tab"
        >
          <div
            class="detailed-information-tab__title"
          >
            Chance of Rain
          </div>
          <div
            class="detailed-information-tab__data"
          >
            12%
          </div>
        </div>
        <div
          class="detailed-information-tab"
        >
          <div
            class="detailed-information-tab__title"
          >
            Humidity
          </div>
          <div
            class="detailed-information-tab__data"
          >
            60%
          </div>
        </div>
        <div
          class="detailed-information-tab"
        >
          <div
            class="detailed-information-tab__title"
          >
            Wind speed
          </div>
          <div
            class="detailed-information-tab__data"
          >
            10 k/h
          </div>
        </div>
        <div
          class="detailed-information-tab"
        >
          <div
            class="detailed-information-tab__title"
          >
            Visibility
          </div>
          <div
            class="detailed-information-tab__data"
          >
            100km
          </div>
        </div>
        <div
          class="detailed-information-tab"
        >
          <div
            class="detailed-information-tab__title"
          >
            Pressure
          </div>
          <div
            class="detailed-information-tab__data"
          >
            1015 hPa
          </div>
        </div>
      </div>
    </div>
    <div
      class="other-days-weather"
    >
      <div
        class="container"
      >
        <div
          class="weather-table"
        >
          <div
            class="weather-table-item--title"
          >
            <div
              class="weather-table-item__container"
            >
              <div
                class="weather-table-item__data-title"
              >
                Day
              </div>
              <div
                class="weather-table-item__data-title"
              >
                Chance of rain
              </div>
              <div
                class="weather-table-item__data-title"
              >
                Humidity
              </div>
              <div
                class="weather-table-item__data-title"
              >
                Wind
              </div>
              <div
                class="weather-table-item__data-title"
              >
                Temperature
              </div>
            </div>
            <hr />
          </div>
          <div
            class="weather-table-item"
          >
            <div
              class="weather-table-item__container"
            >
              <div
                class="weather-table-item__data-title"
              >
                Mon
              </div>
              <div
                class="weather-table-item__data-title"
              >
                9%
              </div>
              <div
                class="weather-table-item__data-title"
              >
                57%
              </div>
              <div
                class="weather-table-item__data-title"
              >
                10 km/h
              </div>
              <div
                class="weather-table-item__data-title"
              >
                19˚
              </div>
            </div>
            <hr />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Render App components should render the Navbar component 1`] = `
<div>
  <div
    class="App"
  >
    <div
      class="navbar"
    >
      <div
        class="container"
      >
        <div
          class="logotype"
        >
          <img
            alt="logotype"
            src="logotype.svg"
          />
        </div>
        <div
          class="search"
        >
          <div
            class="search-field"
          >
            <input
              class="search-bar"
              type="text"
              value=""
            />
            <button
              class="button"
              type="button"
            >
              <img
                alt="loopa"
                src="loopa.svg"
              />
            </button>
          </div>
        </div>
        <button
          class="metrics"
          type="button"
        >
          <span
            class="active"
          >
            °C
          </span>
          <span
            style="margin: 0px 5px;"
          >
            /
          </span>
          <span>
            °F
          </span>
        </button>
      </div>
    </div>
    <div
      class="today-weather"
    >
      <div
        class="container"
      >
        <div
          class="average-data"
        >
          <div
            class="average-data__city"
          />
          <div
            class="average-data__time"
          >
            undefined undefined
          </div>
          <div
            class="average-data__temperature"
          >
            °C
          </div>
        </div>
        <div
          class="general-data"
        >
          <div
            class="data-item"
          >
            <div
              class="data-item__time"
            >
              08:00
            </div>
            <div
              class="data-item__weather-type"
            >
              <img
                alt="cloudy"
                src="cloud.svg"
              />
            </div>
            <div
              class="data-item__temperature"
            >
              13 °C
            </div>
          </div>
          <div
            class="data-item"
          >
            <div
              class="data-item__time"
            >
              09:00
            </div>
            <div
              class="data-item__weather-type"
            >
              <img
                alt="cloudy"
                src="cloud.svg"
              />
            </div>
            <div
              class="data-item__temperature"
            >
              13 °C
            </div>
          </div>
          <div
            class="data-item"
          >
            <div
              class="data-item__time"
            >
              10:00
            </div>
            <div
              class="data-item__weather-type"
            >
              <img
                alt="cloudy"
                src="cloud.svg"
              />
            </div>
            <div
              class="data-item__temperature"
            >
              13 °C
            </div>
          </div>
          <div
            class="data-item"
          >
            <div
              class="data-item__time"
            >
              11:00
            </div>
            <div
              class="data-item__weather-type"
            >
              <img
                alt="cloudy"
                src="cloud.svg"
              />
            </div>
            <div
              class="data-item__temperature"
            >
              13 °C
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="detailed-now-weather"
    >
      <div
        class="detailed-now-weather__hint"
      >
        Now
      </div>
      <div
        class="container"
      >
        <div
          class="detailed-information-tab"
        >
          <div
            class="detailed-information-tab__title"
          >
            Chance of Rain
          </div>
          <div
            class="detailed-information-tab__data"
          >
            12%
          </div>
        </div>
        <div
          class="detailed-information-tab"
        >
          <div
            class="detailed-information-tab__title"
          >
            Humidity
          </div>
          <div
            class="detailed-information-tab__data"
          >
            60%
          </div>
        </div>
        <div
          class="detailed-information-tab"
        >
          <div
            class="detailed-information-tab__title"
          >
            Wind speed
          </div>
          <div
            class="detailed-information-tab__data"
          >
            10 k/h
          </div>
        </div>
        <div
          class="detailed-information-tab"
        >
          <div
            class="detailed-information-tab__title"
          >
            Visibility
          </div>
          <div
            class="detailed-information-tab__data"
          >
            100km
          </div>
        </div>
        <div
          class="detailed-information-tab"
        >
          <div
            class="detailed-information-tab__title"
          >
            Pressure
          </div>
          <div
            class="detailed-information-tab__data"
          >
            1015 hPa
          </div>
        </div>
      </div>
    </div>
    <div
      class="other-days-weather"
    >
      <div
        class="container"
      >
        <div
          class="weather-table"
        >
          <div
            class="weather-table-item--title"
          >
            <div
              class="weather-table-item__container"
            >
              <div
                class="weather-table-item__data-title"
              >
                Day
              </div>
              <div
                class="weather-table-item__data-title"
              >
                Chance of rain
              </div>
              <div
                class="weather-table-item__data-title"
              >
                Humidity
              </div>
              <div
                class="weather-table-item__data-title"
              >
                Wind
              </div>
              <div
                class="weather-table-item__data-title"
              >
                Temperature
              </div>
            </div>
            <hr />
          </div>
          <div
            class="weather-table-item"
          >
            <div
              class="weather-table-item__container"
            >
              <div
                class="weather-table-item__data-title"
              >
                Mon
              </div>
              <div
                class="weather-table-item__data-title"
              >
                9%
              </div>
              <div
                class="weather-table-item__data-title"
              >
                57%
              </div>
              <div
                class="weather-table-item__data-title"
              >
                10 km/h
              </div>
              <div
                class="weather-table-item__data-title"
              >
                19˚
              </div>
            </div>
            <hr />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Render App components should render the dailyWeather component 1`] = `
<div>
  <div
    class="App"
  >
    <div
      class="navbar"
    >
      <div
        class="container"
      >
        <div
          class="logotype"
        >
          <img
            alt="logotype"
            src="logotype.svg"
          />
        </div>
        <div
          class="search"
        >
          <div
            class="search-field"
          >
            <input
              class="search-bar"
              type="text"
              value=""
            />
            <button
              class="button"
              type="button"
            >
              <img
                alt="loopa"
                src="loopa.svg"
              />
            </button>
          </div>
        </div>
        <button
          class="metrics"
          type="button"
        >
          <span
            class="active"
          >
            °C
          </span>
          <span
            style="margin: 0px 5px;"
          >
            /
          </span>
          <span>
            °F
          </span>
        </button>
      </div>
    </div>
    <div
      class="today-weather"
    >
      <div
        class="container"
      >
        <div
          class="average-data"
        >
          <div
            class="average-data__city"
          />
          <div
            class="average-data__time"
          >
            undefined undefined
          </div>
          <div
            class="average-data__temperature"
          >
            °C
          </div>
        </div>
        <div
          class="general-data"
        >
          <div
            class="data-item"
          >
            <div
              class="data-item__time"
            >
              08:00
            </div>
            <div
              class="data-item__weather-type"
            >
              <img
                alt="cloudy"
                src="cloud.svg"
              />
            </div>
            <div
              class="data-item__temperature"
            >
              13 °C
            </div>
          </div>
          <div
            class="data-item"
          >
            <div
              class="data-item__time"
            >
              09:00
            </div>
            <div
              class="data-item__weather-type"
            >
              <img
                alt="cloudy"
                src="cloud.svg"
              />
            </div>
            <div
              class="data-item__temperature"
            >
              13 °C
            </div>
          </div>
          <div
            class="data-item"
          >
            <div
              class="data-item__time"
            >
              10:00
            </div>
            <div
              class="data-item__weather-type"
            >
              <img
                alt="cloudy"
                src="cloud.svg"
              />
            </div>
            <div
              class="data-item__temperature"
            >
              13 °C
            </div>
          </div>
          <div
            class="data-item"
          >
            <div
              class="data-item__time"
            >
              11:00
            </div>
            <div
              class="data-item__weather-type"
            >
              <img
                alt="cloudy"
                src="cloud.svg"
              />
            </div>
            <div
              class="data-item__temperature"
            >
              13 °C
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="detailed-now-weather"
    >
      <div
        class="detailed-now-weather__hint"
      >
        Now
      </div>
      <div
        class="container"
      >
        <div
          class="detailed-information-tab"
        >
          <div
            class="detailed-information-tab__title"
          >
            Chance of Rain
          </div>
          <div
            class="detailed-information-tab__data"
          >
            12%
          </div>
        </div>
        <div
          class="detailed-information-tab"
        >
          <div
            class="detailed-information-tab__title"
          >
            Humidity
          </div>
          <div
            class="detailed-information-tab__data"
          >
            60%
          </div>
        </div>
        <div
          class="detailed-information-tab"
        >
          <div
            class="detailed-information-tab__title"
          >
            Wind speed
          </div>
          <div
            class="detailed-information-tab__data"
          >
            10 k/h
          </div>
        </div>
        <div
          class="detailed-information-tab"
        >
          <div
            class="detailed-information-tab__title"
          >
            Visibility
          </div>
          <div
            class="detailed-information-tab__data"
          >
            100km
          </div>
        </div>
        <div
          class="detailed-information-tab"
        >
          <div
            class="detailed-information-tab__title"
          >
            Pressure
          </div>
          <div
            class="detailed-information-tab__data"
          >
            1015 hPa
          </div>
        </div>
      </div>
    </div>
    <div
      class="other-days-weather"
    >
      <div
        class="container"
      >
        <div
          class="weather-table"
        >
          <div
            class="weather-table-item--title"
          >
            <div
              class="weather-table-item__container"
            >
              <div
                class="weather-table-item__data-title"
              >
                Day
              </div>
              <div
                class="weather-table-item__data-title"
              >
                Chance of rain
              </div>
              <div
                class="weather-table-item__data-title"
              >
                Humidity
              </div>
              <div
                class="weather-table-item__data-title"
              >
                Wind
              </div>
              <div
                class="weather-table-item__data-title"
              >
                Temperature
              </div>
            </div>
            <hr />
          </div>
          <div
            class="weather-table-item"
          >
            <div
              class="weather-table-item__container"
            >
              <div
                class="weather-table-item__data-title"
              >
                Mon
              </div>
              <div
                class="weather-table-item__data-title"
              >
                9%
              </div>
              <div
                class="weather-table-item__data-title"
              >
                57%
              </div>
              <div
                class="weather-table-item__data-title"
              >
                10 km/h
              </div>
              <div
                class="weather-table-item__data-title"
              >
                19˚
              </div>
            </div>
            <hr />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
